src/cider/exec/nextgen/operators/RowToColumnNode.cpp

```diff
@@ -186,6 +154,38 @@ class ColumnWriter {
     }
   }
 
+  JITValuePointer setNullBuffer(JITValuePointer& null_val) {
+    // the null_buffer, raw_data_buffer not used anymore.
+    // so it doesn't matter whether null_buffer is nullptr
+    // or constant false.
+    auto null_buffer = JITValuePointer(nullptr);
+    if (!expr_->get_type_info().get_notnull()) {
+      // TBD: Null representation, bit-array or bool-array.
+      null_buffer.replace(context_.getJITFunction()->createLocalJITValue(
+          [this]() { return allocateBitwiseBuffer(0); }));
+
+      context_.getJITFunction()->emitRuntimeFunctionCall(
+          "set_null_vector_bit",
+          JITFunctionEmitDescriptor{
+              .ret_type = JITTypeTag::VOID,
+              .params_vector = {{null_buffer.get(), index_.get(), null_val.get()}}});
+    } else {
+      null_buffer.replace(context_.getJITFunction()->createLocalJITValue(
+          [this]() { return allocateBitwiseBuffer(0); }));
+
+      context_.getJITFunction()->emitRuntimeFunctionCall(
+          "set_null_vector_bit",
+          JITFunctionEmitDescriptor{
+              .ret_type = JITTypeTag::VOID,
+              .params_vector = {{null_buffer.get(),
+                                 index_.get(),
+                                 context_.getJITFunction()
+                                     ->createConstant(JITTypeTag::BOOL, false)
+                                     .get()}}});
```

Comment:
Hmmm, I think it's better to write a memset runtime function and set null buffer with 0 in local variable block, rather than set it bit-by-bit inside the process loop, which is inefficient. BTW, I forget why all output columns should be allocated a null buffer?😂 Any cases will break if we don't do it?